easybuild/easyconfigs/k/kalign2/kalign2-2.0.4-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,32 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'ConfigureMake'
+
+name = 'kalign2'
```

Comment:
```suggestion
name = 'Kalign'
```

Reply:
That came from [here](https://github.com/kalininalab/alphafold_non_docker)
`conda install -y -c bioconda hmmer==3.3.2 hhsuite==3.3.0 kalign2==2.04` 
Looking at [here](https://msa.sbc.su.se/cgi-bin/msa.cgi) I am actually wondering if we should name that `kalign3` to distinguish between the newer and older versions. However, the GitHub page of that version only reads `kalign`.

Reply:
The reasons to distinguish the names is to allow them to be loaded simultaneously. As far as I can tell, there names of the binaries aren't distinguished so these should conflict. 

Also, we use the capitalization that of the project uses, we shouldn't follow what conda felt like changing it to?

Reply:
Sounds fair to me. Thanks for your guidance here!